### PR TITLE
Add commas after e.g. because American English

### PR DESCRIPTION
**Pull request by SaschaMann**
cc @junedev 
ref https://github.com/exercism/javascript/pull/1160#discussion_r655739622

### CODE REVIEW

**Comment by SaschaMann** [comment]
I hate how this looks but apparently that's just how AE is.

**Comment by junedev** [comment]
> I hate how this looks but apparently that's just how AE is.

Same!

**Comment by iHiD** [comment]
I think this might be beyond the pale for me, and we'll just explicitly state that we allow either form. I can cope with American spellings but I'm not sure I'm willing to cope with this atrocity 🙂

File: building/markdown/style-guide.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Existing descriptions and test-cases can be updated to adhere to them without re
 
 ### Consistency within an exercise
 
-There are some terms that have multiple valid spellings (e.g. "lower case" vs "lowercase").
+There are some terms that have multiple valid spellings (e.g., "lower case" vs "lowercase").
```
Comment (SaschaMann) [inline building/markdown/style-guide.md]:
```suggestion
There are some terms that have multiple valid spellings (e.g. "lower case" vs "lowercase").
```

File: building/markdown/style-guide.md [hunk 2]
```diff
@@ -66,7 +66,9 @@ Some abbreviations are considered common, useful, and non-technical enough that
 - `etc.` or `etc`
 - `docs`
 
-Contractions (e.g. "won't", "I'm", "that's") should be used sparingly if at all in exercise descriptions, but are not restricted in other language around the site (e.g. website copy, mentoring).
+Contractions (e.g., "won't", "I'm", "that's") should be used sparingly if at all in exercise descriptions, but are not restricted in other language around the site (e.g., website copy, mentoring).
```
Comment (SaschaMann) [inline building/markdown/style-guide.md]:
```suggestion
Contractions (e.g. "won't", "I'm", "that's") should be used sparingly if at all in exercise descriptions, but are not restricted in other language around the site (e.g. website copy, mentoring).
```

File: building/markdown/style-guide.md [hunk 3]
```diff
@@ -66,7 +66,9 @@ Some abbreviations are considered common, useful, and non-technical enough that
...
+
+The abbreviations "i.e." and "e.g." should be followed by a comma in American English (see, e.g., this [StackExchange thread](https://english.stackexchange.com/questions/7946/use-of-e-g-are-parentheses-necessary/93658#93658)).
```
Comment (SaschaMann) [inline building/markdown/style-guide.md]:
```suggestion
While many American English style guides state that the abbreviations "i.e." and "e.g." should be followed by a comma in American English (see, e.g., this [StackExchange thread](https://english.stackexchange.com/questions/7946/use-of-e-g-are-parentheses-necessary/93658#93658)), this is not required in text on Exercism".
```

**Review by SaschaMann — COMMENTED** [review]
Thanks <3

File: building/markdown/style-guide.md (hunk 3 above)
Comment (SaschaMann) [reply building/markdown/style-guide.md]: I wanted to add "as per overrule by the British BDFL" but that goes a bit far, I think :P

File: building/markdown/style-guide.md [hunk 4]
```diff
@@ -68,6 +68,8 @@ Some abbreviations are considered common, useful, and non-technical enough that
 
 Contractions (e.g. "won't", "I'm", "that's") should be used sparingly if at all in exercise descriptions, but are not restricted in other language around the site (e.g. website copy, mentoring).
 
+Many American English style guides state that the abbreviations "i.e." and "e.g." should be followed by a comma (see, e.g., this [StackExchange thread](https://english.stackexchange.com/questions/7946/use-of-e-g-are-parentheses-necessary/93658#93658)). This is permitted, but not required within text on Exercism".
```
Comment (SaschaMann) [inline building/markdown/style-guide.md:71]:
```suggestion
Many American English style guides state that the abbreviations "i.e." and "e.g." should be followed by a comma (see, e.g., this [StackExchange thread](https://english.stackexchange.com/questions/7946/use-of-e-g-are-parentheses-necessary/93658#93658)). This is permitted, but not required within text on Exercism.
```

Comment (iHiD) [reply building/markdown/style-guide.md:71]: Dammit.

**Pull request merged**